service-commands/config/convict.json

```diff
@@ -49,7 +49,7 @@
   },
   "user_node": {
     "env": "USER_NODE",
-    "default": "http://cn1_creator-node_1:4000"
+    "default": "http://cn-um_creator-node_1:4099"
```

Comment:
I think you need to add this domain name to `etc/hosts/` for this to work

Reply:
whoops! just realized that this PR includes that change. 

may not be relevant for this PR, but `sudo node hosts.js add` does not work with the new um mapping depending on the user's local /etc/hosts state. I personally added some mappings, and this command checks for the line difference VS actual content. @piazzatron 

in my local test, I manually added the um mapping

Reply:
yeah same. this would need to be run by everyone unless we can run remove and then add works

Reply:
remove and add does work though, but maybe we can add that as part of the local setup by default each time?

Reply:
like when you run node setup up it runs hosts down/up?

Reply:
not sure if i like that? i want to be certain about changes to hosts file. we dont change it enough to warrant auto update i feel like

Reply:
alternative is that when this releases everyone will inevitably miss any msg we share out and ask questions repeatedly. i don't really feel strongly about it

Reply:
discussed offline -- I interpreted the add method wrong, and @SidSethi will add a comment to run remove and then add for the /etc/hosts file to be updated properly :)

Reply:
added!